src/main/java/Game.java

```diff
@@ -21,13 +21,20 @@ public boolean hasPlayerWonSet(Players playerWhoIsWinning, Players playerWhoIsLo
             return true;
 
         } else if (playerWhoIsWinning.getSetScore() >= 11 && playerWhoIsLoosing.getSetScore() >= 10) {
-            return hasSomeOneOfThePlayersTwoPointsMore();
+            return hasSomeoneOfPlayersTwoPointsMore();
 
         } else {
             return false;
         }
     }
 
+    public boolean whichPlayerSide(Players playerOne, Players playerTwo) {
```

Comment:
All this function does is calculating whether the match score is even or not. It technically doesn't say anything about which player is playing on which side of the table.